taggit/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig as BaseConfig
+from django.utils.translation import ugettext_lazy as _
+
+class AppConfig(BaseConfig):
```

Comment:
The standard around App config is to name the config class based on the module name. For example:

 ```python
 # rock_n_roll/apps.py

from django.apps import AppConfig

class RockNRollConfig(AppConfig):
    name = 'rock_n_roll'
    verbose_name = "Rock ’n’ roll"
```
From the [docs](https://docs.djangoproject.com/en/1.9/ref/applications/#for-application-authors).